feat(curriculum): add steps for employee profile generator workshop (#64896)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/blocks/workshop-employee-profile-generator/694bf6eb530e19a9c48c59f4.md

---
id: 694bf6eb530e19a9c48c59f4
title: Step 13
challengeType: 20
dashedName: step-13
---

# --description--

Concatenating many strings using `+` and converting numbers using `str()` can get messy and hard to read.

Python 3.6 introduced **f-strings** to solve this. By adding the letter `f` before the opening quote, you can put variables and expressions inside replacement fields represented represented by curly braces `{}`. For example:

```python
name = 'John'
print(f'Hello {name}') # Output: Hello John
```

Create a variable `employee_card` and assign it an f-string that displays `Employee:` followed by a space and the value of the `full_name` variable.

# --hints--

> 

Reviewer: ````suggestion

You should have a variable named `employee_card`.

```js
({ test: () => assert(runPython(`_Node(_code).has_variable("employee_card")`)) })
```

````

---
id: 694bf6eb530e19a9c48c59f4
title: Step 13
challengeType: 20
dashedName: step-13
---

# --description--

Concatenating many strings using `+` and converting numbers using `str()` can get messy and hard to read.
Python 3.6 introduced **f-strings** to solve this. By adding the letter `f` before the opening quote, you can put variables directly inside curly braces `{}`.

Example:

```python
name = 'John'
print(f'Hello {name}') # Output: Hello John
```

Create a variable `employee_card` and assign it an f-string that displays `Employee:` followed by the value of the `full_name` variable.

# --hints--

You should create `employee_card` with the correct string.

```js

> ({ test: () => assert(runPython(`_Node(_code).has_variable("employee_card")`)) })

Reviewer 2: you are only checking the existence of the variable, not what is assigned to it, the hint and the test do not match

---
id: 694bf6eb530e19a9c48c59f4
title: Step 13
challengeType: 20
dashedName: step-13
---

# --description--

Concatenating many strings using `+` and converting numbers using `str()` can get messy and hard to read.

Python 3.6 introduced **f-strings** to solve this. By adding the letter `f` before the opening quote, you can put variables directly inside curly braces `{}`.

Example:

```python
name = 'John'
print(f'Hello {name}') # Output: Hello John
```

Create a variable `employee_card` and assign it an f-string that displays `Employee:` followed by the value of the `full_name` variable.

# --hints--

> You should create `employee_card` with the correct string.

Reviewer: ```suggestion
Your `employee_card` variable should have the value `f'Employee: {full_name}'`.
```
